tools/nvqpp/nvq++.in

```diff
@@ -311,6 +311,24 @@ TARGET_ARGS=
 if [[ $# -eq 0 ]]; then
     SHOW_HELP=true
 fi
+# Handle arguments with '='
+N=$#
```

Comment:
nit: not used

Reply:
Used on line#324, to check for last argument